route/pool.go

```diff
@@ -156,9 +157,10 @@ type EndpointPool struct {
 	NextIdx            int
 	maxConnsPerBackend int64
 
-	random    *rand.Rand
-	logger    logger.Logger
-	updatedAt time.Time
+	random      *rand.Rand
+	logger      logger.Logger
+	updatedAt   time.Time
+	LBAlgorithm string
```

Comment:
Can we please use consistent naming? The property is spelled out at all other places.

Reply:
I have selected this name to provide consistency to the EndpointPool structure field naming convention, e.g. another property is called maxConnsPerBackend. Naming it LoadBalancingAlgorithm would stand out as the Java styled name in the structure. I see your point though and will consider to rename it.